control-plane/.gitignore

```diff
@@ -0,0 +1,8 @@
+node_modules
```

Comment:
Here is a good example of a `.gitignore` for Firebase projects: 
https://github.com/GoogleChromeLabs/trust-safety-demo/blob/main/conversion-measurement/.gitignore